charts/sourcegraph/CHANGELOG.md

```diff
@@ -16,7 +16,7 @@ All notable changes to Sourcegraph are documented in this file.
 
 ### Changed
 
--
+- **CAUTION** Migrated Code Insights from Timescale to Postgres. If you have customized the configmap or connection parameters for codeinsights-db, manual updates may be required. [#74](https://github.com/sourcegraph/deploy-sourcegraph-helm/pull/74)
```

Comment:
Full details on this will be provided in the Kubernetes upgrade docs. We'll need to think about whether we create Helm-specific upgrade docs too or just refer to this changelog.